src/Arcus.Templates.WebApi/Startup.cs

```diff
@@ -67,8 +88,38 @@ public void ConfigureServices(IServiceCollection services)
 #endif
 #if CertificateAuth
                 options.Filters.Add(new CertificateAuthenticationFilter());
+#endif
+#if JwtAuth
+                AuthorizationPolicy policy = 
+                    new AuthorizationPolicyBuilder()
+                        .RequireRole("Admin")
+                        .RequireAuthenticatedUser()
+                        .Build();
+
+                options.Filters.Add(new AuthorizeFilter(policy));
 #endif
             });
+#if JwtAuth
+            var key = Encoding.UTF8.GetBytes(Configuration["Jwt:SigningKey"]);
```

Comment:
Can't we get that from the ISecretProvider since it's a key?

Reply:
Yeah, that would be cool. The startup isn't async though, not sure if we could do that. Hmm.

Reply:
We should do that, regardless if it's ideal because it's a secret 🤷‍♂